Numerics/EvaluatingFunctions/remez.py

```diff
@@ -0,0 +1,144 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# This python script implements Remez' algorithm.
+# To use the script, you can modify what happens when this script is executed:
+# See the `if __name__ == "__main__":` branch near the end of this file.
+#
+# Alternatively, `runRemez` can be imported and run directly, see the docstring
+# of runRemez.
+
+import math
+import numpy as np
+from matplotlib import pyplot as plt
+
+# Return n chebyshev nodes on the interval (a,b)
+def _getChebyshevNodes(n, a, b):
```

Comment:
PEP 8 style would suggest renaming as `_snake_case`:
```suggestion
def _get_chebyshev_nodes(n, a, b):
```